MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -502,13 +505,12 @@ void DumpWord(Quad * pq) {
 }
 )                                       /* End of debug code */
 
-_Unchecked
 void DumpWords(void) {
 static int X;
   int i;
   X = (X+1) & 1023;
   if (X != 0) return;
-    for (i = 0; i < cpwLast; i++) _Unchecked{ wprint((char*)apwSol[i]->pchWord); }
+    for (i = 0; i < cpwLast; i++) _Unchecked { wprint((_Nt_array_ptr<char>)apwSol[i]->pchWord); }
```

Comment:
Then this won't be either

Reply:
Oh it has an unchecked block because we couldn't prove the bounds anyway.

Reply:
Yes, that's why I didn't mention it.